pkg/proxy/ipvs/proxier.go

```diff
@@ -1357,31 +1366,43 @@ func (proxier *Proxier) syncProxyRules() {
 			}
 
 			// Build ipvs kernel routes for each node ip address
-			nodeIPs, err := proxier.ipGetter.NodeIPs()
+			nodeIPs := make([]net.IP, 0)
+			addresses, err := utilproxy.GetNodeAddresss(proxier.nodePortAddresses, proxier.networkInterface)
 			if err != nil {
-				glog.Errorf("Failed to get node IP, err: %v", err)
+				glog.Errorf("Failed to get node ip address matching nodeport cidr")
```

Comment:
Shouldn't we bail out instead of having the else?

```go
if ... {
   glog.Errorf(...)
   return
}
...
```

Reply:
Done.